src/shared/models/order.model.ts

```diff
@@ -0,0 +1,14 @@
+import { orderItems } from "./order-items.model";
+import { User } from "./user.model";
+
+export interface Order {
+    _id?: string,
+    orderItems: orderItems[],
+    shippingAddress1: string,
```

Comment:
the shape of shippingAddress property can also be of **shippingAddress1: ShippingAddress** where 'ShippingAddress' can be an interface in the same model which holds properties like city, zip, country, phone inside it

Reply:
I 100% agree with Aman on this one tho lol.. stop being lazy

Reply:
I honestly didn't even think to do that good idea lol